pygmt/tests/test_config.py

```diff
@@ -72,8 +72,13 @@ def test_config_format_date_map():
 
     Note the space in 'o dd', this acts as a regression test for
     https://github.com/GenericMappingTools/pygmt/issues/247.
+
+    Setting FORMAT_DATE_MAP="yyyy mm dd" as a regression test for
+    https://github.com/GenericMappingTools/pygmt/issues/2298.
     """
     fig = Figure()
+    # Set FORMAT_DATE_MAP to "yyyy mm dd" which contains whitespaces.
+    config(FORMAT_DATE_MAP="yyyy mm dd")
```

Comment:
Is a test being added to use this whitespace-containing `FORMAT_DATE_MAP` or is this sufficient to be used on other tests to make sure that whitespace can work?

Reply:
> is this sufficient to be used on other tests to make sure that whitespace can work?

Yes, please see https://github.com/GenericMappingTools/pygmt/issues/2298#issuecomment-1396555512